backend/src/logger/settings.ts

```diff
@@ -23,9 +25,29 @@ export function getSourceLocationTrackingSetting(): boolean {
   return config.get(SOURCE_TRACKING_CONFIG_PROP);
 }
 
+/**
+ * @returns {string}
+ */
+export function getTargetFolderSetting(): string {
+  const config = vscode.workspace.getConfiguration();
+  return config.get(TARGET_FOLDER_CONFIG_PROP);
+}
+
+/**
+ * @returns {string}
+ */
+export function getSelectedWorkspaceSetting(): string {
+  const config = vscode.workspace.getConfiguration();
+  return config.get(SELECTED_WORKSPACE_CONFIG_PROP);
+}
+
```

Comment:
/**
 * @returns {string}
 */
export function getSelectedWorkspaceSetting(): string {
  return this.getSetting(SELECTED_WORKSPACE_CONFIG_PROP);
}
/**
 * @returns {string}
 */
private function getSetting(section: string): string {
  return vscode.workspace.getConfiguration().get(section);
}